components/org.wso2.carbon.identity.application.authz.xacml/src/test/java/org.wso2.carbon.identity.application.authz.xacml/handler.impl/XACMLBasedAuthorizationHandlerTest.java

```diff
@@ -0,0 +1,50 @@
+package org.wso2.carbon.identity.application.authz.xacml.handler.impl;
```

Comment:
Please add Licensee header and class defenition

Reply:
done